python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -767,6 +767,13 @@ def __clone_lr_op_sub_block__(op, program, lr_block):
             prefetch_var_name_to_block_id.extend(
                 lookup_table_var_name_to_block_id)
 
+        if len(optimize_blocks) == 0:
+            pre_block_idx = pserver_program.num_blocks - 1
+            empty_block = pserver_program._create_block(pre_block_idx)
+            optimize_blocks.append(empty_block)
```

Comment:
Need a warning?

Reply:
Why we need empty pserver block here? how about just raise an exception, additional pservers may let trainers to send unnecessary data and reduce the total performance